GoogleChromeLabs/ps-analysis-tool
# Feature: Embed explorable explanations in PSAT extension

## Description
This PR aims to embed explorable explanation in PSAT extension.

## Relevant Technical Choices
- Webpack in production mode now gives a dist file from Explorable Explanations package.
- This package exports `userSketch`, `interestGroupSketch` and main sketch which is imported by the extension package.

## Testing Instructions
- Clone this branch.
- In the terminal run `npm i && npm run build:ext`
- Now go to example.com and open the DevTools.
- In DevTools navigate to the PSAT extension.
- Go to Protected Audience tab and select explorable explanations you should now see the animation.

## Additional Information:



## Screenshot/Screencast


https://github.com/user-attachments/assets/b4352d38-1b5c-4e39-ab47-cca911d77871


---

## Checklist






Partially addresses #857
